charts/bitwarden-crd-operator/Chart.yaml

```
@@ -95,8 +95,8 @@ annotations:
   artifacthub.io/operator: "true"
   artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/changes: |
-    - kind: fixed
-      description: "Fixed bitwarden installation"
+    - kind: feat
```

```suggestion
    - kind: added
```
supported kinds are added, changed, deprecated, removed, fixed and security as stated here: https://artifacthub.io/docs/topics/annotations/helm/